src/main/java/com/github/wovnio/wovnjava/WovnHttpServletResponse.java

```diff
@@ -80,4 +80,25 @@ public void setHeader(String name, String value) {
         }
         super.setHeader(name, value);
     }
+
+    public void flushBuffer() throws IOException {
+        flush();
```

Comment:
Call `flush()` is need before call `super.flushBuffer()`.

But when disabled `super.flushBuffer()`,
that's not needed.
How about moving `flush()` to into `if (this.headers.settings.enableFlushBuffer) {`?

Reply:
> But when disabled `super.flushBuffer()`, that's not needed.

Is this true though?

The behavior of `writer.flush()` and `output.flush()` depends on the implementation of the underlying stream. In our particular case it actually does nothing, because the underlying stream is ultimately an `OutputStream`.

But if the implementation of `FilterServletOutputStream` was different, and flush wrote buffered content, I think it would make sense to do that whenever the wrapped servlet calls `flushBuffer()`.

Alternatively, I can actually remove `flush()` completely.

Reply:
Of course, currently `flush()` do nothing. So no problem.

I just concern below behavior could be a problem in the future.

Programmer can guess why nothing happen when call `flushBuffer()` because of `enableFlushBuffer` is disabled. But actually, call `flush()` and does not call `super.flushBuffer()`.

It's also hard to guessing without code reading. 


But it's not big thing. I approve  it even if no change.